spec/support/project_hepler.rb

```diff
@@ -0,0 +1,6 @@
+def add_project
+  visit root_path
+  click_link 'Projects'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.